packages/igx-templates/igx-ts/custom-templates/awesome-grid/files/src/app/__path__/__filePrefix__.component.ts

```diff
@@ -99,6 +106,26 @@ export class <%=ClassName%>Component implements OnInit, OnDestroy {
     }
   }
 
+  public getSpeed(athlete: any): any {
+    athlete['Speed'] = this.getSpeedeData(40);
```

Comment:
Lint doesn't like this type of assignment and shows errors
```suggestion
    athlete.Speed = this.getSpeedeData(40);
```